ee/tabby-db/src/notifications.rs

```diff
@@ -33,7 +33,13 @@ impl DbConn {
 
     pub async fn mark_notification_read(&self, id: i64, user_id: i64) -> Result<()> {
         query!(
-            "INSERT INTO read_notifications (notification_id, user_id) VALUES (?, ?)",
+            "INSERT INTO read_notifications (notification_id, user_id)
```

Comment:
you might use `ON CONFLICT` query to handle this situation:

https://www.sqlite.org/lang_upsert.html